### ISSUE TITLE
If a hololens 2 goes to sleep while running a Unity OpenXR MRTK app with spatial awareness enabled, the app throws exception and runs poorly after resuming

### ISSUE THREAD

**Issue by larryNexus** [microsoft/MixedRealityToolkit-Unity#10244]
## Describe the bug

A hololens 2 app built in Unity using OpenXR with spatial awareness runs properly on launch until the device goes to sleep. After waking the device and resuming the app, the app throws this exception:

```
ArgumentNullException: Value cannot be null.
Parameter name: _unity_self
```

at 

```
UnityEngine.XR.XRMeshSubsystem.SetBoundingVolume(UnityEngine.Vector3 origin, UnityEngine.Vector3 extents).
```

(See screenshot below for a more complete stack trace)

The application also runs very poorly from that point forward, at a frame-rate of approximately 15fps accoring the the MRTK diagnostic tool

## To reproduce

1. Create an MRTK app in Unity for hololens 2 with spatial awareness enabled by following the MRTK project setup instructions  [here](https://docs.microsoft.com/en-us/learn/modules/learn-mrtk-tutorials/1-3-exercise-configure-unity-for-windows-mixed-reality).
2. Switch to UWP build platform, then build and run the app on hololens 2.
3. See that spatial awareness and the spatial mesh are running properly
4. Put the hololens 2 device to sleep
5. Wake the hololens 2 device and resume running the app
6. See that the app throws the above exception and is running very poorly

To make it easy to reproduce this error, I have created a nearly-empty MRTK project repository on github.

The only modifications to this project as compared to the MRTK tutorial instructions are:
* Cloned the default hololens 2 MRTK configuration to enable spatial mesh and set mesh visibility to occlusion
* Added an ErrorManager that receives any exceptions thrown and displays them on a camera-anchored canvas

The repository for that project is [here](https://github.com/larryNexus/mrtk-testing).

## Expected behavior

When the hololens 2 device goes to sleep, the application should pause. When the hololens 2 is woken up and the app the app is resumed, the app should function properly, as it did on launch, without throwing any exceptions.

## Screenshots

See attached screenshot of the exception thrown, rendered to an in-app text canvas.
![HololensMRTKErrorAfterResume](https://user-images.githubusercontent.com/91710257/135660356-460e2769-732f-48e4-864c-e7c22ba7ea28.jpg)

## Your setup (please complete the following information)

- Unity Version: 2020.3.17 and 2020.3.8
- MRTK Version: 2.7.2
- MRTK OpenXR: 1.0.1, 1.0.3
- com.unity.xr.openxr 1.2.8

## Target platform (please complete the following information)

- HoloLens 2, ARM or ARM64

**Comment by polar-kev** [comment]
Will route this to the OpenXR team for further investigation.

**Comment by srinjoym** [comment]
@polar-kev Any updates on this issue? I am hitting this exception when resuming from sleep in our apps as well

**Comment by keveleigh** [comment]
@srinjoym Update! This should be fixed by #10366, which is currently targeting 2.7.3 (FYI @polar-kev)

**Comment by keveleigh** [comment]
Closing this out now that the fix is merged into the 2.7.3 branch.
